### ISSUE TITLE
TypeError: __init__() got multiple values for argument 'cookie_directory'

### ISSUE THREAD

**Issue by yi-yunseok** [icloud-photos-downloader/icloud_photos_downloader#602]
## Overview
icloudpd, version 1.10.0, `icloudpd -d ~/Pictures -a album`
`TypeError: __init__() got multiple values for argument 'cookie_directory'`

## Steps to Reproduce
- Run icloudpd to download specific album

## Expected Behavior
icloudpd downloads album typed.

## Actual Behavior
Can't download with `TypeError` in `Authenticating`

## Context
```sh
2023-03-12 14:21:46 DEBUG    Authenticating...
Traceback (most recent call last):
  File "/usr/local/bin/icloudpd", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/icloudpd/base.py", line 288, in main
    core(
  File "/usr/local/lib/python3.9/dist-packages/icloudpd/base.py", line 571, in core
    icloud = authenticator(domain)(
  File "/usr/local/lib/python3.9/dist-packages/icloudpd/authentication.py", line 31, in authenticate_
    icloud = pyicloud_ipd.PyiCloudService(
TypeError: __init__() got multiple values for argument 'cookie_directory'
```

**Comment by AndreyNikiforov** [comment]
@yi-yunseok Can you try binary package from GitHub Release section, please? Post command you are executing in addition to the output too.

**Comment by yi-yunseok** [comment]
Ok I will soon.

**Comment by yi-yunseok** [comment]
Solved with icloudpd 1.12.0. Thank you.
